bsp/resources/messages/ScalaBspBundle.properties

```diff
@@ -37,6 +37,8 @@ bsp.resolver.resolving.build.structure.failed=resolving BSP build structure fail
 bsp.resolver.import.failed=import failed
 bsp.resolver.creating.bloop.configuration.from.sbt=creating Bloop configuration from sbt
 bsp.resolver.sources=sources
+bsp.resolver.jvmenvironment=jvm environment
```

Comment:
capitalize JVM

Reply:
actually we didn't add new endpoint so I should just delete it